### PR TITLE
Remove image-related Client Hints

### PR DESCRIPTION
**Pull request by yoavweiss**
As previously discussed, the current state where some CH hints are defined in the IETF draft and some in accompanying specifications is confusing. Furthermore, defining the image related hints in web specifications would be a better fit, as they have direct implications to image processing there.

This PR removes those hints from the IETF draft. This hints are [being added](https://github.com/yoavweiss/html/commit/7bd134ec67472303d06700686be5d37d45ada048) to the HTML specification as part of a larger Client Hints related PR.

### CODE REVIEW

File: draft-ietf-httpbis-client-hints.md [hunk 1]
```diff
@@ -93,9 +93,9 @@ This document defines a new response header, Accept-CH, that allows an origin se
 
 Client Hints mitigate the performance concerns by assuring that clients will only send the request headers when they're actually going to be used, and the privacy concerns of passive fingerprinting by requiring explicit opt-in and disclosure of required headers by the server through the use of the Accept-CH response header.
 
-This document also defines an initial set of Client Hints.
+This document defined the Client Hints infrastructure, but does not define any specific features that will use it. Those features will be defined in their respective specifications.
```
Comment (igrigorik) [inline draft-ietf-httpbis-client-hints.md]: s/defined/defines? Past tense doesn't make sense here, I think?

Is "infrastructure" well understood term, or should we elaborate a bit on what this means? Something like, a framework for advertising hints that the client wants to receive, persisting them, and corresponding server logic? Ish?

File: draft-ietf-httpbis-client-hints.md [hunk 2]
```diff
@@ -133,12 +132,12 @@ Servers can advertise support for Client Hints using the Accept-CH header field
 For example:
 
 ~~~ example
-  Accept-CH: DPR, Width, Viewport-Width
+  Accept-CH: Example-CH-Hint, Example-CH-Hint-2
```
Comment (igrigorik) [inline draft-ietf-httpbis-client-hints.md]: Add `Sec-` prefix? `Sec-CH-Example1`, ...

File: draft-ietf-httpbis-client-hints.md (hunk 1 above)
Comment (yoavweiss) [reply draft-ietf-httpbis-client-hints.md]: > s/defined/defines? Past tense doesn't make sense here, I think?

Oops. definitely "defines".

I'll also elaborate on "infrastructure"

File: draft-ietf-httpbis-client-hints.md (hunk 2 above)
Comment (yoavweiss) [reply draft-ietf-httpbis-client-hints.md]: yup

**Comment by igrigorik** [comment]
Merging. 

@mnot if you have any nits or suggestions, please chime in here and we can address in a separate PR. In the meantime, I'll push a new draft so we have the updated version to discuss at upcoming IETF.

**Pull request merged**